sys/net/gnrc/routing/ipv6_auto_subnets/gnrc_ipv6_auto_subnets.c

```diff
@@ -429,13 +490,14 @@ static void _configure_subnets(uint8_t subnets, uint8_t start_idx, gnrc_netif_t
 
     /* immediately send an RA with RIO */
     if (ext_opts) {
-        gnrc_ndp_rtr_adv_send(upstream, NULL, NULL, true, ext_opts);
+        gnrc_ndp_rtr_adv_send(upstream, NULL, src, true, ext_opts);
     } else {
         DEBUG("auto_subnets: Options empty, not sending RA\n");
     }
 }
 
-void gnrc_ipv6_nib_rtr_adv_pio_cb(gnrc_netif_t *upstream, const ndp_opt_pi_t *pio)
+void gnrc_ipv6_nib_rtr_adv_pio_cb(gnrc_netif_t *upstream, const ndp_opt_pi_t *pio,
+                                  const ipv6_addr_t *src)
```

Comment:
Is this an API breaking change that needs to be labeled as such?

Reply:
At least I would expect a change in the corresponding header documentation.

Reply:
So far this has been an internal API that isn't documented in any public header